net/net_appsock.h

```diff
@@ -43,7 +43,7 @@ extern int gbl_libevent_appsock;
 #define check_appsock_rd_thd() check_thd(appsock_rd_thd)
 #define check_appsock_timer_thd() check_thd(appsock_timer_thd)
 
-#define event_once(a, b, c)                                                    \
+#define evtimer_once(a, b, c)                                                    \
```

Comment:
I renamed event_once to evtimer_once as libevent has its own [event_once](http://www.wangafu.net/~nickm/libevent-2.0/doxygen/html/event__compat_8h.html#a074cf31709035ec2b7b660374bbd3b90).